<launch>
    <param name="distanceCtrl/P" type="double" value="1.0"/>
    <param name="distanceCtrl/I" type="double" value="0.0"/>
    <param name="distanceCtrl/D" type="double" value="0.0"/>

    <param name="angleCtrl/P" type="double" value="1.0"/>
    <param name="angleCtrl/I" type="double" value="0.0"/>
    <param name="angleCtrl/D" type="double" value="0.1"/>

    <param name="angleRateCtrl'clean_angleCtrl/P" type="double" value="0.13"/>
    <param name="angleRateCtrl'clean_angleCtrl/I" type="double" value="0.0"/>
    <param name="angleRateCtrl'clean_angleCtrl/D" type="double" value="0.0"/>
</launch>